fearless/Modules/AccountImport/Model/MnemonicTextProcessor.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import SoraFoundation
+
+struct MnemonicTextProcessor: TextProcessing {
+    func process(text: String) -> String {
+        let cuttedTabAndLineBreaks = text
+            .replacingOccurrences(of: "\n", with: "")
+            .replacingOccurrences(of: "\t", with: "")
+        var result = cuttedTabAndLineBreaks
+        while result.contains("  ") {
```

Comment:
Can we just split by CharacterSet.whitespaces, trim and then filter out empty elements?